docs/MTX.md

```diff
@@ -15,23 +15,26 @@ Step 1. is beyond the scope of this article; please consult the project's discus
 
 Note: An ArduPilot flight controller is assumed. For PX4 it needs to be tested and seen. INAV won't work AFAIK, as INAV is not a proper MAVLink component.
 
-## OpenTx Radio Setup
+## OpenTx/mTX Radio Setup
 
 - MDL->MODEL SETUP->External RF->Mode = mBridge
 
-For more details on other possible configurations, please consult the MAVLink for OpenTx project discussion channels.
+For more details on other possible configurations, please consult the mTX project discussion channels.
 
 ## mLRS Tx Module Setup
 
 - Tx Ch Source = mbridge
-- Tx Ser Dest = mbridge
-- Tx Snd RadioStat = off
 
-Note: There are situations in which it can be useful to enable "Tx Snd RadioStat", but you should do this only if you know what you are doing.
+Note: The parameter "Tx Snd RadioStat" should normally be set to off. There are situations in which it can be useful to enable "Tx Snd RadioStat", but you should do this only if you know what you are doing.
+
+Note: The parameter "Tx Ser Dest" can be set to serial, serial2 or mbridge. If serial or serial2 is selected, then a MAVLink router is enabled which routes the messages between serial or serial2, the mTX radio, and the mLRS link. This allows you to e.g. in addition connect a GCS to serial or serial2. If mbridge is selected, then a MAVLink router is not invoked and the MAVLink communication happens only between the mTX radio and the mLRS link.   
```

Comment:
Slight edit:

'This allows you to connect a GCS to serial or serial2. If mbridge is selected, then the MAVLink router is not enabled and the MAVLink communication happens only between the mTX radio and the mLRS link.'

Reply:
THX. done.